pkg/volume/volume.go

```diff
@@ -334,6 +334,13 @@ func (v *Operation) Read() (*v1alpha1.CASVolume, error) {
 		map[string]interface{}{
 			string(v1alpha1.OwnerVTP):        v.volume.Name,
 			string(v1alpha1.RunNamespaceVTP): v.volume.Namespace,
+			string(v1alpha1.IsPatchVolumePropertyVTP): func() string {
```

Comment:
This is great way to think & implement functional logic. Good Job @utkarshmani1997 :+1: